src/functions/findFirst.js

```diff
@@ -78,6 +93,32 @@ export const findFirst = wrap({
         }
         return undefined;
     },
+    tests: process.env.NODE_ENV !== "development" ? undefined : {
+        "basicUsage": (hi) => {
+            const seq = hi("a string to be searched").findFirst("to be");
+            hi.assert(seq, (r) => r.index === 9);
```

Comment:
There should also be a similar example for case-insensitive substring search e.g.

``` js
const asciiCaseInsensitive = (a, b) => (a.toUpperCase() === b.toUpperCase());
const found = hi("hello world").findFirst("World", asciiCaseInsensitive);
hi.assert(found.index === 6);
hi.assert(found.string() === "world");
```